### ISSUE TITLE
TensorFlow-Slim

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#200]
#22 
> TF-Slim是tensorflow中用来定义、训练与评估的轻量级库，tf-slim的组件与tf.contrib.learn相似能够将你从复杂的原生tensorflow解放出来。

# Reference
- [ ] [TensorFlow-Slim](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/slim)
- [ ] [TensorFlow-Slim image classification model library](https://github.com/tensorflow/models/tree/master/research/slim)
- [ ] [TensorFlow-slim 训练 CNN 分类模型](https://www.jianshu.com/p/84c0e2f7f1a5)
- [ ] [TensorFlow-Slim API -黑暗星球](https://blog.csdn.net/u014061630/article/details/80632736)

# Components

- [ ] arg_scope: provides a new scope named arg_scope that allows a user to define default arguments for specific operations within that scope.
- [ ] data: contains TF-slim's dataset definition, data providers, parallel_reader, and decoding utilities.
- [ ] evaluation: contains routines for evaluating models.
- [ ] layers: contains high level layers for building models using tensorflow.
- [ ] learning: contains routines for training models.
- [ ] losses: contains commonly used loss functions.
- [ ] metrics: contains popular evaluation metrics.
- [ ] nets: contains popular network definitions such as VGG and AlexNet models.
- [ ] queues: provides a context manager for easily and safely starting and closing QueueRunners.
- [ ] regularizers: contains weight regularizers.
- [ ] variables: provides convenience wrappers for variable creation and manipulation.
